internal/website/content/howto/pubsub/subscribe.md

```diff
@@ -7,48 +7,73 @@ toc: true
 
 Subscribing to messages on a topic with the Go CDK takes three steps:
 
-1. Open the subscription with the Pub/Sub provider of your choice (once per subscription).
+1. Open the subscription with the Pub/Sub provider of your choice (once per
```

Comment:
Make these links to the subsequent sections like in the other file?

Reply:
Done, and changed to 2 steps as discussed